recipes/omero-rdf/LICENSE.txt

```diff
@@ -0,0 +1,339 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is in the tarball. You don't need to add it here again.

Reply:
Done, thanks!

Reply:
I may be missing something, but the file is still here, no?